src/uct/ib/configure.m4

```diff
@@ -239,9 +239,17 @@ AS_IF([test "x$with_ib" == xyes],
        AC_CHECK_DECLS([IBV_LINK_LAYER_INFINIBAND,
                        IBV_LINK_LAYER_ETHERNET,
                        IBV_EVENT_GID_CHANGE,
-                       ibv_create_qp_ex],
+                       ibv_create_qp_ex,
+                       ibv_create_srq_ex],
                       [], [], [[#include <infiniband/verbs.h>]])
 
+       AC_CHECK_DECL(ibv_query_device_ex, [
+       AC_TRY_COMPILE([#include <infiniband/verbs.h>],
```

Comment:
why AC_CHECK_DECLS/AC_CHECK_FUNC is not enough?

Reply:
In some old MOFED (2.1) ibv_query_device_ex() has different prototype, two arguments.

Reply:
ok, probably need to add comment